tools/pipelines/templates/include-publish-npm-package-deployment.yml

```
@@ -51,28 +51,46 @@ jobs:
   variables:
     version: $[ stageDependencies.build.build.outputs['SetVersion.version']]
     isLatest: $[ stageDependencies.build.build.outputs['SetVersion.isLatest']]
+  templateContext:
+    type: releaseJob
+    isProduction: ${{ not(eq(variables['testBuild'], 'true')) }}
```

The condition on the `testBuild` variable would consider CI builds for main as production builds. We have a `release` variable that can be `"release"`, `"prerelease"`, or blank, so I think this check is preferable.
```suggestion
    # in() returns true if the first parameter is equal to any of the others
    isProduction: ${{ in(variables['release'], 'release', 'prerelease')  }}
```

---

Updated.